tests/metagpt/document_store/test_qdrant_store.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+@Time    : 2023/6/11 21:08
+@Author  : alexanderwu
+@File    : test_milvus_store.py
+"""
+import random
+
+from qdrant_client.models import PointStruct, Distance, VectorParams, FieldCondition, Range, Filter
+
+from metagpt.document_store.qdrant_store import QdrantConnection, QdrantStore
+from metagpt.logs import logger
+
+vectors = [[random.random() for _ in range(2)] for _ in range(10)]
+
+points = [
+    PointStruct(
+        id=idx,
+        vector=vector,
+        payload={"color": "red", "rand_number": idx % 10}
```

Comment:
In vector-store scenarios, people not only focus on vectors but also on how to convey meta information such as text. Can you add this in the unit test or example.